devtools/src/arcs-bug-report.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {PolymerElement} from '../deps/@polymer/polymer/polymer-element.js';
+import {html} from '../deps/@polymer/polymer/lib/utils/html-tag.js';
+import {MessengerMixin} from './arcs-shared.js';
+import '../deps/jszip/dist/jszip.js';
+
+/**
+ * Saves/loads a bug report which contains debug messages sent to devtools.
+ */
+class ArcsBugReport extends MessengerMixin(PolymerElement) {
+  static get template() {
+    return html`
+    <style include="shared-styles">
+      :host {
+        display: flex;
+        flex-direction: column;
+      }
+    </style>
+    <div class="wrapper">
+      <input type="file" id="fileElem" style="display:none" on-change="_loadBugReport">
+      <button id="fileSelect">Load Bug Report</button>
```

Comment:
This UI will only be visible when Arcs Explorer is already connected to some Arcs Runtime, otherwise if Arcs Explorer is not connected to anything you will see something like:

![Screen Shot 2019-09-24 at 10 47 31 am](https://user-images.githubusercontent.com/26159485/65536662-babfee00-deb8-11e9-88be-5010b090ae56.png)

(this is rendered by arcs-connection-status FWIW)

The right thing to do would be to upload the Bug Report _instead_ of having a live connection, so I would imagine having something like:

![Screen Shot 2019-09-24 at 10 50 27 am](https://user-images.githubusercontent.com/26159485/65536918-23a76600-deb9-11e9-946a-5797575783cd.png)

Now if you upload the bug report this Arcs Explorer instance would no longer wait for Arcs Shell, actually it should disconnect from the ALDS entirely, so that another Arcs Explorer can be used to exploration of the live shell.

Another option that may be easier is to have a URL parameter `?bugreport` that is handled like other connection types in devtools.js and will cause opening up a file picker instead of the websocket or webrtc connection. This "Load a Bug Report" link could just link to "href=?bugreport", then it would all just work without much work I think...

WDYT?

Reply:
Adding the parameter sgtm.